iOS/DuckDuckGo/Refactoring/Default/DefaultOmniBarView.swift

```diff
@@ -257,7 +259,9 @@ final class DefaultOmniBarView: UIView {
 
     override func layoutSubviews() {
         super.layoutSubviews()
-        NotificationCenter.default.post(name: DefaultOmniBarView.didLayoutNotification, object: self)
```

Comment:
This is here to allow DuckPlayer know the Omnibar position.  We're using a notification as a quick win now, but will be moved to using UserDefauts notifications